src/FileList/FileListHandler.cc

```diff
@@ -12,6 +12,7 @@
 #include <casacore/casa/OS/File.h>
 
 #include "FileInfoLoader.h"
+#include "Logger/Logger.h"
```

Comment:
Is the intention here to replace the fmt include? See previous comment about the location of the fmtlib headers bundled with spdlog.

Reply:
This also applies to the other include changes.